jetty-server/src/main/config/modules/server.mod

```diff
@@ -91,3 +91,7 @@ etc/jetty.xml
 # jetty.scheduler.name=
 # jetty.scheduler.deamon=false
 # jetty.scheduler.threads=-1
+
+# Whether to use direct ByteBuffers for reading or writing
+jetty.httpConfig.useInputDirectByteBuffers=true
+jetty.httpConfig.useOutputDirectByteBuffers=true
```

Comment:
These should be commented out like the rest of the lines, and description should probably start with two `##` like the rest of them.

Maybe this should also be put directly under the other `jetty.httpConfig` setters.

Reply:
correct. fixed (arghh we don't have checkstyle configured for mod file 😄)